### DISCUSSION TITLE
Will the transfer of filecoins record the IP address?

### DISCUSSION THREAD

**Discussion by ChenPi-1122** [filecoin-project/community#192] [Q&A]
I have a question Will the transfer of coins record the IP address?Any one who can tell me?thks

**Comment by longfeiWan9** [comment]
Interesting question, but I do not think FIL transferring message/transaction will record your IP address. But it will definitely record your wallet address (something like f1*******) or ID address (something like f0****).

- You can refer to a FIL transfer transaction on FileScan for more detail, [link is here](https://filscan.io/tipset/message-detail?cid=bafy2bzacebijxvj2axqxvtunifjcqonafwq3l7yermqpwlqpnndxht7elrs2i)

**Comment by jimpick** [comment]
Your IP address won't be recorded on the blockchain. However, you will need to sign a message (typically with a wallet app like the online Glif wallet), and then that will need to send the message to a Filecoin node that will broadcast it to the message pool so it can be recorded to the blockchain by a miner. A web based wallet or Filecoin node might possibly log your IP address for their own purposes, so make sure you use one that you trust.

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
even if not using a "service" but you own node to broadcast the message to the p2p network - someone watching it closely might be able to record IP addresses messages originated from. the proper obfuscation techniques are recommended as with any other IP based traffic on the "internet" to, at least, make it harder to trace back your message. proxies, vpn, tor....
